ote/src/cljs/ote/app/controller/transport_service.cljs

```diff
@@ -77,6 +77,10 @@
 (defrecord EnsureCsvFileResponse [response])
 (defrecord FailedCsvFileResponse [response])
 
+(defrecord EnsureUrl [url])
```

Comment:
These are specific to external interface URLs... maybe they should be renamed to be something like EnsureExternalInterfaceUrl